tests/bridge.py

```diff
@@ -7,21 +7,25 @@
 class TestBridge(unittest.TestCase):
 
     def setUp(self):
-        self.link = bench.links[0]
-        self.link.host_if.flushAddresses()
-        self.link.host_if.addAddress("192.168.10.2/24")
+        links = bench.links
+
+        for i, l in enumerate(links, start=1):
+            l.host_if.flushAddresses()
+
         self.bridge = bench.target.addBridge("br0")
         self.bridge.up()
-
+        self.bridge.addAddress("192.168.10.1/24")
 
     def tearDown(self):
-        self.link.host_if.delAddress("192.168.10.2/24")
         self.bridge.down()
         bench.target.delBridge(self.bridge)
 
 
     def test_add_interface_to_bridge(self):
-        self.bridge.addInterface(self.link.target_if)
-        self.bridge.addAddress("192.168.10.1/24")
-        self.link.host_if.ping("192.168.10.1", count=1, deadline=1)
+        links = bench.links
 
+        for i, l in enumerate(links, start=1):
```

Comment:
Ditto, enumerate is useless if you don't use `i`. It would actually be cool to be able to verify that `ping` fails at first, and succeeds when the interface is added to the bridge. But there are no clean ways to do that (yet!).